frontend/src/components/metadata/CreateMetadataDefinition.tsx

```diff
@@ -0,0 +1,115 @@
+import React, {useEffect, useState} from "react";
+
+import {Box, Button, Grid, TextField} from "@mui/material";
+import {useDispatch, useSelector,} from "react-redux";
+import {RootState} from "../../types/data";
+
+import TopBar from "../navigation/TopBar";
+import { postMetadataDefinitions } from "../../actions/metadata";
+
+export const CreateMetadataDefinitionPage = (): JSX.Element => {
+	return (
+		<div>
+			<TopBar/>
+			<CreateMetadataDefinition/>
+		</div>
+	);
+}
+
+export const CreateMetadataDefinition = (): JSX.Element => {
+
+	const dispatch = useDispatch();
+	// @ts-ignore
+	const saveMetadataDefinitions = (metadata: object) => dispatch(postMetadataDefinitions(metadata));
+
+	const [metadataName, setMetadataName] = React.useState<string>("");
+	const [metadataDescription, setMetadataDescription] = React.useState<string>("");
+
+	const addNewField = () => {
+		// TODO: Complete this portion
+	}
+
+    const postMetadata = () => {
+		console.log("Inside postMetadata");
+
+		// TODO: Make the field populate dynamically
+		saveMetadataDefinitions({
+			"name" : metadataName,
+			"description" : metadataDescription,
+			"context" : {
+				"doi" : "https://schema.org/DigitalDocument"
+			},
+			"fields" : [
+				{
+					"name" : "doi",
+					"list" : false,
+					"widgetType": "TextField",
+					"config": {
+						"type" : "str"
+					},
+					"required" : true
+				}
+			]
+		});
+
+		clearForm();
+	}
+
+	const clearForm = () => {
+		setMetadataName("");
+		setMetadataDescription("");
+	}
+
+	return (
+		<div className="outer-container">
+			<div className="inner-container">
+				<Grid container spacing={2} rowSpacing={4}>
+					<Grid xs={12} md={8}>
+						<TextField
```

Comment:
You can use styled clowder text field this way it will take the full width
See `frontend/src/components/styledComponents/ClowderInput.jsx` or `frontend/src/components/styledComponents/ClowderMetadataTextField.jsx`

Reply:
<img width="657" alt="image" src="https://user-images.githubusercontent.com/13950475/194582554-d074f626-ef95-4424-84f8-2a5816b5194e.png">
 
 right now the input/form didn't take the full width of the dialog

Reply:
Sorry for nitpicking... the cancel on the bottom right seems a little bit odd (?). Maybe "x" on the top right might make more sense?